examples/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
-cmake_minimum_required(VERSION 3.0...3.16)
-project(OpenNS_Examples LANGUAGES C CXX)
+cmake_minimum_required(VERSION 3.15)
```

Comment:
why do we need multiple instance of 'cmake_minimum_required'? isn't enough to declare it in the root's CMakeLists?